tests/integration/tests/services_test.go

```diff
@@ -37,6 +37,30 @@ func TestServicesList(t *testing.T) {
 	require.Equal(kiali.BOOKINFO, serviceList.Namespace.Name)
 }
 
+func TestClusterServicesList(t *testing.T) {
+	require := require.New(t)
+	serviceList, err := kiali.ClusterServicesList(kiali.BOOKINFO)
+
+	require.NoError(err)
+	require.NotEmpty(serviceList)
+	require.True(len(serviceList.Services) >= 4)
```

Comment:
It would probably be good to first query the kube api for services and then query the kiali api for services and make sure the two list lengths match. I'm not sure the significance of `>= 4` here.

Reply:
this is the copy paste of previous test but including the new API call

Reply:
there should be at least 4 default bookinfo services